### PR TITLE
V0.804.0

### PR DESCRIPTION
**Pull request by muelli**
this history wants to be cleaned up. But because it doesn't build yet, we keep it around.

### CODE REVIEW

**Comment by muelli** [comment]
it builds. whee :)

@flub I'd appreciate if you can have a quick look.

I've tested the built version and it works fine for me.

File: Cargo.lock [hunk 1]
```diff
@@ -0,0 +1,2626 @@
+# This file is automatically @generated by Cargo.
```
Comment (flub) [inline Cargo.lock]: How come we have to check in this file?  Isn't it included in the deltachat-core-rust repo already?  Or is this a different file?

File: chat.delta.desktop.yml [hunk 2]
```diff
@@ -102,46 +130,52 @@ modules:
...
 
       # All the generated npm downloads, see README.md how to create
-      # this file.
-      # This needs to go to the bottom so that the commands in that file can see package.json and friends.
-      - generated-sources.json
+      # this file.  This needs to go to the bottom so that the
+      # commands in that file can see package.json and friends.
+      - generated-sources-npm.0.json
+      - generated-sources-npm.1.json
+      - generated-sources-npm.2.json
```
Comment (flub) [inline chat.delta.desktop.yml]: How come there's 3 of these now?  The readme doesn't explain that part.

File: Cargo.lock (hunk 1 above)
Comment (muelli) [reply Cargo.lock]: good catch.

File: chat.delta.desktop.yml (hunk 2 above)
Comment (muelli) [reply chat.delta.desktop.yml]: I've added a paragraph to the readme. Thanks for catching this!

File: README.md [hunk 3]
```diff
@@ -64,23 +80,30 @@ beforehand.  This is done by letting npm put the dependencies into a
 converting that into a manifest snipped called
 `generated-sources.json`.
 
-You need to generate the `package-lock.json` file from the
-deltachat-desktop release you need.  Go to the git repository,
-chechout the correct tag and run (e.g. using docker):
+Upstream ships the package-lock.json file so it should not be
+necessary to generate it.  However, sometimes the file is not
+updated in lockstep with package.json and then dependencies will
+be missing during build time.
```
Comment (flub) [inline README.md]: Is this why that file is still in this repository?

Comment (muelli) [reply README.md]: yeah. in the manifest is a reference to the bug report.

**Comment by flub** [comment]
Hey, is it intentional to do this before you merged #24? I think it
contains useful improvements, but maybe you had some issues with it?

**Pull request merged**